### ISSUE TITLE
grpc-js not emitting error on failed client

### ISSUE THREAD

**Issue by onepiers** [grpc/grpc-node#1411]
### Problem description
We migrated over from grpc to grpc-js and now experience a different behaviour in reconnect scenarios. Based on our experience with grpc a client that once errors due to the server not being available was able to trigger new calls and tried again until the service was responsive again. This has changed with grpc-js. Once the client is not able to reach the server we get: 
Error: 14 UNAVAILABLE: No connection established and all sub-sequential calls result in no error on the client call.

### Reproduction steps
1. Create a client and keep the same instance in a service
2. Call the server when it is offline
3. Call it again in intervals
4. All calls after the first do not emmit events (end,data,error)

### Environment
 - OS name, version and architecture: mac os 10.15.2 (19C57)
 - Node version v12.8.1
 - Node installation method yarn
 - Package name and version grpc-js 1.0.1

**Comment by murgatroid99** [comment]
Can you share some code that demonstrates this problem?

**Comment by onepiers** [comment]
```js
export class ClientProvider {
  private readonly stub: EventServiceClient;

  constructor(private readonly configService: ConfigService) {
    const backendUrl = "..."
    this.stub = new EventServiceClient(backendUrl, grpc.credentials.createInsecure());
// -> grpc.makeClientConstructor(..)
  }

  getClient(): EventServiceClient {
    return this.stub;
  }
}

export class Connector {
  private readonly stub: EventServiceClient;

  constructor(private readonly provider: ClientProvider) {
    this.stub = provider.getClient();
  }

  getEvents(startingId: number): Observable<EventResponse[]> {
   //...
    const call = this.stub.getEvents(request);
   // these will never get called after failing once
    call
      .on('data', (event: EventResponse) => {
        eventQueue.push(event);
      })
      .on('end', () => {
        subject.next(eventQueue);
        subject.complete();
      })
      .on('error', (e: Error) => {
        subject.error(e);
      });
    return subject;
  }
}
```

**Comment by murgatroid99** [comment]
To be clear, you are saying that if you call `getEvents` and it fails, then later calls to `getEvents` result in nothing happening?

**Comment by onepiers** [comment]
Yes exactly. After getting an error 14 UNAVAILABLE: No connection established after the first call nothing else is happening on the next calls.

**Comment by murgatroid99** [comment]
I can reproduce this problem. Thank you for the bug report.

**Comment by murgatroid99** [comment]
You can probably work around this problem by using the `round_robin` load balancing policy. See https://github.com/grpc/grpc-node/issues/1307#issuecomment-601312421 for an explanation of how to enable it.

**Comment by onepiers** [comment]
Sadly that did not help, we'll wait for the fix.

**Comment by murgatroid99** [comment]
I published a change that appears to fix the problem in grpc-js version 1.0.4. Can you update to that version and try it out?
